src/myr/reference.js

```diff
@@ -253,6 +253,11 @@ let transformations = [
         description: <span>The setColor function changes the color of the cursor. For general colors, <a href='https://htmlcolorcodes.com/color-names/' target="_blank" rel="noopener noreferrer">HTML color codes</a> can be used (e.g., "blue", "deeppink"), and for specific colors HEX colors can be used (e.g., "#00ffff"). The default cursor state is "red".</span>,
         example: "setColor"
     },
+    {
+        name: "setTexture",
+        parameters: [{type: "string", name: "texture"}],
+        description: <span>The setColor function changes the texture of the cursor. The unaltered texture is displayed when the color is white, otherwise the texture will be affected by the current color.</span>,
```

Comment:
How will an end user know our texture options?

Reply:
Is there anywhere you think it would be best to list them? Textures will now autocomplete which should also help.